src/components/interaction-designer/resource-editors/ResourceVariantTextEditor.vue

```diff
@@ -31,16 +31,13 @@
         </template>
       </span>
 
-      <tooltip
+      <b-tooltip
           v-if="doesContentContainExpression"
-          :content="`<p>${trans('flow-builder.youre-using-floip-expressions')}</p>
+          v-b-tooltip.hover.top.html="`<p>${trans('flow-builder.youre-using-floip-expressions')}</p>
```

Comment:
@safydy I think this directive is supposed to be on the `<a>` tag, then we don't need to wrap it in a `<b-tooltip>`! I could be wrong though, I'm referencing: https://bootstrap-vue.org/docs/directives/tooltip